app/assets/stylesheets/accessible-autocomplete.scss

```diff
@@ -0,0 +1,7 @@
+.clear-search {
+  margin-top: 50px;
+
+  &-error {
+    margin-top: 90px;
+  }
+}
```

Comment:
RWD: would it be ideal to use the [govuk spacing mixin and values](https://design-system.service.gov.uk/styles/spacing/#applying-spacing-in-your-own-css)?

Reply:
The [values only go up to 60px](https://design-system.service.gov.uk/styles/spacing/#static-spacing), sadly

I could do something funky like have the 50px always there and add the extra 40px when there's an error, by either using variables or adding the margin to two separate elements, but they didn't seem right. What do you think?

Reply:
Nothing funky.
Funky is not standard practice and it always becomes a pain.

As the max value is only 60px. I would suggest we test the responsiveness of your units.